web/src/redux/testOutputs/slice.ts

```diff
@@ -0,0 +1,43 @@
+import {createSlice, PayloadAction} from '@reduxjs/toolkit';
+
+import {TTestOutput} from 'types/TestOutput.types';
+
+interface ITestOutputsState {
+  initialOutputs: TTestOutput[];
+  outputs: TTestOutput[];
+}
+
+const initialState: ITestOutputsState = {
+  initialOutputs: [],
+  outputs: [],
+};
+
+const testOutputsSlice = createSlice({
+  name: 'testOutputs',
+  initialState,
+  reducers: {
+    outputsInitiated(state, action: PayloadAction<TTestOutput[]>) {
```

Comment:
Redux strong recommendation: [Model Actions as Events, Not Setters](https://redux.js.org/style-guide/#model-actions-as-events-not-setters)

Reply:
Makes sense, but I think as this is just a preference I won't worry too much as it doesn't change the behavior of functionality

Reply:
Yes, agree. However it could change the way you think about actions. Probably right now we are dispatching multiple actions for 1 single event. For example, we dispatch an action to set a value in the store but also to clean something else.